packages/peregrine/lib/talons/RootComponents/Category/categoryContent.gql.js

```diff
@@ -19,10 +19,10 @@ export const GET_PRODUCT_FILTERS_BY_CATEGORY = gql`
 `;
 
 export const GET_CATEGORY_CONTENT = gql`
-    query getCategoryData($id: String!) {
-        categories(filters: { ids: { in: [$id] } }) {
+    query getCategoryData($uid: String!) {
```

Comment:
It is better to keep same variables names on queries, otherwise it will cause a breaking change.